### PR TITLE
Convert src/ to LF eol

### PR DESCRIPTION
**Pull request by Kelebek1**
Some files, mainly Opus, were mistakenly committed with CRLF line endings, causing me some grief with git. I ran dos2unix over src/, so these should be the only outstanding files.

### CODE REVIEW

**Comment by v1993** [comment]
Probably a good idea to create `.git-blame-ignore-revs` and add this commit there.

**Comment by Kelebek1** [comment]
> Probably a good idea to create `.git-blame-ignore-revs` and add this commit there.

Not sure it matters too much in this case, since I did Opus anyway? Is it a problem for the config files?

**Comment by v1993** [comment]
Well, having a cleaner blame is always nice, but use this as you see fit.

**Comment by Kelebek1** [comment]
Mmk, I added it anyway, I'm not sure how it works if I try to merge these commits together, and then ignore the commit that adds the ignore file, so I'll leave them separate.

**Comment by v1993** [comment]
You can't have them as a single commit since amending a commit changes its hash. Looks like `reuse` is unhappy about this file, though.

File: .git-blame-ignore-revs [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# SPDX-FileCopyrightText: 2018 yuzu Emulator Project
```
Comment (liamwhite) [inline .git-blame-ignore-revs:1]: $current_year

**Review by liamwhite — APPROVED** [review]

**Pull request merged**